snap/snapcraft.yaml

```diff
@@ -39,7 +38,17 @@ parts:
     build-packages:
       - patchelf
     stage-packages:
-      - libffi6
+      - libffi-dev
```

Comment:
are you certain that the `libffi-dev` package is required here over `libffi6`? It is _very_ unusual to stage any `-dev` packages.

Reply:
Thanks - this is the kind of feedback I needed. `libffi6` doesn't exist in Focal / core20. I've updated it to require `libffi7`.